test/E2ETests/SmokeTestsOnNanoServer.cs

```diff
@@ -263,6 +264,12 @@ public SmokeTestsOnNanoServer(ITestOutputHelper output, RemoteDeploymentConfig c
                     ApplicationBaseUriHint = applicationBaseUrl,
                     ApplicationType = applicationType
                 };
+
+                if (applicationType == ApplicationType.Standalone)
+                {
+                    deploymentParameters.AdditionalPublishParameters = "-r win10-x64";
```

Comment:
I am hardcoding this because NanoServer's RID is for Windows 2016 but our CI servers seem to be either Win 2012 or less (which have a different RID)...NanoServer tests work by publishing to a temp folder on the CI server and then copy bits to the target NanoServer